xray/config_test.go

```diff
@@ -89,46 +88,46 @@ func popEnv(env []string) {
 	}
 }
 
-func ResetConfig() {
```

Comment:
What's the reason for commenting out this function ?

Reply:
same reason as https://github.com/aws/aws-xray-sdk-go/pull/177#discussion_r368185353
and TestDefaultConfigureParameters doesn't touch the environment values.